composer.json

```diff
@@ -13,7 +13,7 @@
         "predis/predis": "^1.0",
         "laravelcollective/html": "^5.5",
         "proengsoft/laravel-jsvalidation": "^1.6.0-dev",
-        "sonarsoftware/customer_portal_framework": ">=1.0.11",
+        "sonarsoftware/customer_portal_framework": "^1.0",
```

Comment:
This needs to be 1.0.12

Reply:
You'll also need to update the lockfile.

Reply:
@geneccx lockfile is already set to 1.0.12. I've made the changes within the composer.json file.